File: 3_Hospital/Sprint3/app/models/TS03/Address.js
module.exports = (sequelize, DataTypes) => {
    const Address = sequelize.define('Address', {
		add_id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
		add_street: {
			type: DataTypes.STRING,
			allowNull: false,
		},
		add_number: { type: DataTypes.INTEGER,
			allowNull: false
		},
		add_city: {
			type: DataTypes.STRING,
			allowNull: false,
		},
		add_state: {
			type: DataTypes.STRING,
			allowNull: false,
		},
		add_country: {
			type: DataTypes.STRING,
			allowNull: false,
		},
		add_zip_code: {
			type: DataTypes.STRING,
			allowNull: false,
		},
	},{
		freezeTableName: true,
		timestamps: false,
		underscored: true,
	});

	Address.associate = function(models){
		Address.hasMany(models.Hospital, 
			{
				foreignKey: 'add_id', 
				onDelete: 'CASCADE', 
				hooks: true 
			}
		);
	};

	return Address;

};

File: 1_Paciente/backend/stepesbd-api/Model/Address.cs
using System;
using System.Collections.Generic;

namespace stepesdb_api
{
    public partial class Address
    {
        public Address()
        {
            Person = new HashSet<Person>();
        }

        public int AddId { get; set; }
        public string AddStreet { get; set; }
        public string AddNumber { get; set; }
        public string AddExtraNumber { get; set; }
        public string AddCity { get; set; }
        public string AddState { get; set; }
        public string AddNeighborhood { get; set; }
        public string AddCountry { get; set; }
        public string AddZipcode { get; set; }

        public virtual ICollection<Person> Person { get; set; }
    }
}
